src/pro/sky/java/course1/HW12/Book.java

```diff
@@ -0,0 +1,29 @@
+package pro.sky.java.course1.HW12;
+
+public class Book {
+    private String name;
+    private Author author;
+    private int year;
+
+    public Book(String name, Author author, int year) {
+        this.author = author;
+        this.name = name;
+        this.year = year;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public String getAuthor() {
```

Comment:
```suggestion
    public String getAuthorFullName() {
```
потому что getAuthor должен возвращать именно объект автора - поле author